src/index.js

```diff
@@ -6,9 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import reportWebVitals from './reportWebVitals';
+import App from './App.js';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById("root")
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Planthor-Team/Planthor_ClientFrontEndWebApp/pullRequest?prid=7095961)
```suggestion
  document.getElementById("root"),
```